shared/src/commonMain/kotlin/co/nimblehq/mark/kmmic/domain/model/User.kt

```diff
@@ -0,0 +1,18 @@
+package co.nimblehq.mark.kmmic.domain.model
+
+import co.nimblehq.mark.kmmic.data.service.user.model.UserResponse
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class User constructor(
```

Comment:
We could omit contructor
```suggestion
data class User(
```

Reply:
@suho updated at [3225037](https://github.com/markgravity/kmm-ic/pull/72/commits/3225037d11388c90ad505fdf1de85c0ac1d7d6f2)